src/validate/validate.ts

```diff
@@ -47,6 +48,7 @@ const VALIDATORS = {
     'sky': validateSky,
     'terrain': validateTerrain,
     'projection': validateProjection,
+    'projection-mode': validateProjectionMode,
```

Comment:
Not sure I fully understand this definition. How would that look in a style json file?

Reply:
```
{
  sky: {}
  terrain: {}
  projection: { 
    mode: ['step', ['zoom'], 'globe', 10, 'mercator']
  }
}
```